pkg/network/server.go

```diff
@@ -295,6 +308,14 @@ func (s *Server) Start() {
 	go s.relayBlocksLoop()
 	go s.bQueue.Run()
 	go s.bSyncQueue.Run()
+	go s.bFetcherQueue.Run()
+	if s.ServerConfig.NeoFSBlockFetcherCfg.Enabled && s.blockFetcher != nil {
+		err := s.blockFetcher.Start()
+		if err != nil {
+			s.log.Error("NeoFS block fetcher service:", zap.Error(err))
```

Comment:
We need to decide to use it everywhere: is it `NeoFS blockfetcher`/`NeoFS BlockFetcher`/`NeoFS block fetcher`. I'd suggest `NeoFS BlockFetcher` because you use this name in the node configuration.

Suggestions?

Reply:
i like `NeoFS BlockFetcher`